pkg/testutil/e2eutil/prometheus.go

```diff
@@ -277,6 +280,134 @@ func (p *Prometheus) Appender() tsdb.Appender {
 	return p.db.Appender()
 }
 
+// CreateEmptyBlock produces empty block like it was the case before fix: https://github.com/prometheus/tsdb/pull/374.
+// (Prometheus pre v2.7.0).
+func CreateEmptyBlock(dir string, blockDelay time.Duration, mint int64, maxt int64, extLset labels.Labels, resolution int64) (ulid.ULID, error) {
```

Comment:
Is `blockDelay` ever not `0`? AFAICT there aren't any current users and everything uses `CreateBlockWithBlockDelay`. Do you plan on using this?

Reply:
What about, 
```suggestion
func CreateEmptyBlock(dir string, creationTime time.Time, mint int64, maxt int64, extLset labels.Labels, resolution int64) (ulid.ULID, error) {
```
? It is more verbose. block delay feels like you want to upload it with delay or something... Let's be explicit.